workspaces/noWMS/namespace.xml

```diff
@@ -0,0 +1,5 @@
+<namespace>
+  <id>NamespaceInfoImpl-459f1a44:1624bc3a9cf:-7ffb</id>
+  <prefix>noWMS</prefix>
```

Comment:
Is this still a good name?

Reply:
I'm thinking the work to hide layers will still be done.